### PR TITLE
#165372941: Fetch tweet feeds on user profile 

### PR DESCRIPTION
**Pull request by gtsofa**
#### What does this PR do?
- Enables a user to see their tweets on a tweet feeds.
#### Description of Task to be completed?
- Fetch tweets from the backend and display them on user profiles.
#### How should this be manually tested?
- Checkout to this branch
- Run `npm install --save`
- Start the serve `npm start`
- Login and visit the home page
#### Any background context you want to provide?
- None
#### What are the relevant PT Stories?
[165372941](https://www.pivotaltracker.com/n/projects/2327271/stories/165372941)

### CODE REVIEW

File: src/actions/tweet.actions.js [hunk 1]
```diff
@@ -28,12 +28,13 @@ export const tweet = (data, history) => dispatch => {
...
-            url, 
-            data, 
+            url,
+            data,
             {headers: headers},
           )
         .then(response => {
-            console.log('hfhghgcvbxgcbcvv', response)
+            // console.log('hfhghgcvbxgcbcvv', response)
+            console.log(response);
```
Comment (susanwere) [inline src/actions/tweet.actions.js]: You could do away with console here

Comment (gtsofa) [reply src/actions/tweet.actions.js]: This was part of your task implementation @susanwere , I forgot to ask you what they were doing. I will remove them then

File: src/containers/Home.jsx [hunk 2]
```diff
@@ -1,114 +1,132 @@
...
-        this.props.tweet(data)
+  handleSubmit = e => {
+    const { tweet, history } = this.props;
+    if (!e.target.checkValidity()) {
+      e.preventDefault();
+      e.stopPropagation();
     }
+    e.preventDefault();
+    const data = { body: this.state.text_value };
+    console.log(data);
```
Comment (susanwere) [inline src/containers/Home.jsx]: console.log kindly remove it

**Review by susanwere — APPROVED** [review]
LGTM. Good job @gtsofa

**Review by maxwellgithinji — APPROVED** [review]
LGTM. Nice work @gtsofa

**Review by wrotich — APPROVED** [review]
LGTM. Good job @gtsofa

**Pull request merged**